config/pipeline0-task.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: pipeline0-task
+spec:
+  inputs:
+    resources:
+    - name: git-source
+      type: git
+    params:
+    - name: pathToResourceFiles
+      description: The path to the resource files to apply
+      default: /workspace/git-source
+    - name: apply-directory
+      description: The directory from which resources are to be applied
+      default: ""
+  steps:
+  - name: kubectl-apply
+    image: lachlanevenson/k8s-kubectl
```

Comment:
Should this be pinned to a specific version?

Reply:
It should always be fine just to use the latest image here as all we are doing is a `kubectl apply`